### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @SebaBranda

### CODE REVIEW

**Comment by SebaBranda** [comment]
@faloi termine, pero no se por que no me toma las imagenes.

File: images/Barrio.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="142px" preserveAspectRatio="none" style="width:328px;height:142px;" version="1.1" viewBox="0 0 328 142" width="328px" zoomAndPan="magnify"><defs><filter height="300%" id="f3q0myvbygkyg" width="300%" x="-1" y="-1"><feGaussianBlur result="blurOut" stdDeviation="2.0"/><feColorMatrix in="blurOut" result="blurOut2" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 .4 0"/><feOffset dx="4.0" dy="4.0" in="blurOut2" result="blurOut3"/><feBlend in="SourceGraphic" in2="blurOut3" mode="normal"/></filter></defs><g><!--MD5=[b983d684e7da15fb3f2ce530bc980bb4]
```
Comment (faloi) [inline images/Barrio.svg:1]: ¿Con SVG no te anduvo el README?

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,23 +1,23 @@
 class Barrio {
 
-	const property elementos = []
+	const property elementos = [] #propiedad de la clase - lista
```
Comment (faloi) [inline src/barrio.wlk:3]: ¿Lista de qué?

File: src/elementos.wlk [hunk 3]
```diff
@@ -1,52 +1,52 @@
 class Hogar {
 
-	var property nivelMugre
-	var property confort
+	var property nivelMugre #propiedades de la clase
+	var property confort #propiedades de la clase
```
Comment (faloi) [inline src/elementos.wlk:4]: La gracia era decir de qué tipo son.

File: src/elementos.wlk [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
 
 	var property capacidadProduccion
 
-	method esBueno() {
+	method esBueno() { #consulta - devuelve un booleano
 		return capacidadProduccion > INTA.produccionMinima()
 	}
 
-	method fueAtacado(plaga) {
+	method fueAtacado(plaga) { #indicacion - no devuelve nada
```
Comment (faloi) [inline src/elementos.wlk:38]: Faltó el tipo del parámetro (aunque sea obvio).

**Pull request closed without merging**